### ISSUE TITLE
component publish  command need support auto updates of version

### ISSUE THREAD

**Issue by yugasun** [serverless/components#724]
# Description

`components publish xxx` command need to support auto updates of version in `serverless.component.yml`, like below:

```
components version patch/minor/major
```

**Comment by tinafangkunding** [comment]
cc @eahefnawy

**Comment by eahefnawy** [comment]
I'm not following. 🤔 


You mean that when you run `sls publish` it should automatically bump the version?

**Comment by yugasun** [comment]
@eahefnawy Yeah, just like `npm version patch`, it will update `version` in `package.json` automatically.
